ramls/samples/profiles/mappingProfileRule.sample

```diff
@@ -0,0 +1,7 @@
+{
+   "id":"electronicAccess.linkText",
```

Comment:
id in folio usually means UUID, this could be a little confusing , can we use some other name. Like fieldName? or something else

Reply:
renamed to the "fieldId"